### DISCUSSION TITLE
Welcome to canboat Discussions!

### DISCUSSION THREAD

**Discussion by keesverruijt** [canboat/canboat#224] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by svstayer** [comment]
Hello ! I use the Ethernet Adapter from Yacht Devices which gives me N2K native data over TCP. Can I configure canboat to use it. On a raspberry pi ?

**Reply by keesverruijt** [reply]
Hi @svstayer the idea would be that you start a new discussion for this unrelated question, but yes, the YD format is supported.

**Comment by tmkahelin** [comment]
I am working on a system for my boat, that is running on a Beckhoff industrial PC PLC, Win CE/Twincat runtime. I am not very experienced with PLC programming, just learning as I go. I am planning to receive NMEA2000 traffic via CanOpen interface to the PLC. Thank you all for the amazing work you have done with this project!
